.gitignore

```diff
@@ -122,6 +122,7 @@ package-lock.json
 # Visual studio
 .vscode
 .vs
+bin/
```

Comment:
Added this line because [vscode-java](https://github.com/redhat-developer/vscode-java) extension creates this folder on Java projects 🙂

Reply:
I have the same problem. Still I won't add `bin/` here (I added it in my global `.gitignore`). 

The problem is on VS Code's end, as they used a folder name which is too generic and potentially used in production. This has the risk of accidentally make us skip artifacts during NPM packaging or git commits.